src/berry_mill/kiwrap.py

```diff
@@ -85,6 +87,17 @@ def add_repo(self, reponame: str, repodata: Dict[str, str]) -> KiwiParent:
 
         return self
 
+    def _verify_gpg_key(self, key_path) -> bool:
+        """
+        Virify wether the downloaded file is a GPG key
+        """
+        try:
+            return bool(subprocess.run(['gpg', '--import', key_path], capture_output=True, text=True).returncode == 0)
```

Comment:
Well, this is really still wrong. You want to return `.returncode` from the `subprocess.run` and then check it, right? 😉 
So:
```python
import os

try:
  return bool(subprocess.run(['gpg', '--import', key_path],
                             capture_output=True, text=True).returncode == os.EX_OK
except Exception as e:
  ...
```

And where is the unit test for this?